types/tests/tsconfig.json

```diff
@@ -8,6 +8,6 @@
   },
   "include": [
     "../index.esm.d.ts",
-    "./**/*.ts",
+    "../../dist/**/*.d.ts"
```

Comment:
Why this change?
If I understand correctly it skips all the ts tests now.

Reply:
I'm guessing this is an issue with typescript module resolution.
Maybe `"moduleResolution": "NodeNext",` in tsconfig would help? (could also bring other issues)

Reply:
I have tried right now.. It doesn't help.

```
npm run test-types

> chartjs-chart-treemap@2.1.3 test-types
> tsc -p types/tests/

node_modules/chart.js/dist/helpers.d.ts:1:23 - error TS2307: Cannot find module '@kurkle/color' or its corresponding type declarations.

1 import { Color } from '@kurkle/color';
                        ~~~~~~~~~~~~~~~

types/index.esm.d.ts:8:8 - error TS1479: The current file is a CommonJS module whose imports will produce 'require' calls; however, the referenced file is an ECMAScript module and cannot be imported with 'require'. Consider writing a dynamic 'import("chart.js")' call instead.

8 } from 'chart.js';
         ~~~~~~~~~~

types/index.esm.d.ts:111:16 - error TS1479: The current file is a CommonJS module whose imports will produce 'require' calls; however, the referenced file is an ECMAScript module and cannot be imported with 'require'. Consider writing a dynamic 'import("chart.js")' call instead.

111 declare module 'chart.js' {
                   ~~~~~~~~~~

types/tests/options.ts:2:23 - error TS1479: The current file is a CommonJS module whose imports will produce 'require' calls; however, the referenced file is an ECMAScript module and cannot be imported with 'require'. Consider writing a dynamic 'import("chart.js")' call instead.
  To convert this file to an ECMAScript module, change its file extension to '.mts', or add the field `"type": "module"` to 'C:/Users/AndreaStocchero/git/chartjs-chart-treemap/package.json'.

2 import { Chart } from 'chart.js';
                        ~~~~~~~~~~

types/tests/options.ts:3:35 - error TS1479: The current file is a CommonJS module whose imports will produce 'require' calls; however, the referenced file is an ECMAScript module and cannot be imported with 'require'. Consider writing a dynamic 'import("chart.js/helpers")' call instead.
  To convert this file to an ECMAScript module, change its file extension to '.mts', or add the field `"type": "module"` to '/git/chartjs-chart-treemap/package.json'.

3 import { color as colorLib } from 'chart.js/helpers';
                                    ~~~~~~~~~~~~~~~~~~


Found 5 errors in 3 files.

Errors  Files
     1  node_modules/chart.js/dist/helpers.d.ts:1
     2  types/index.esm.d.ts:8
     2  types/tests/options.ts:2

```

Reply:
@kurkle I didn't find a way...

Nevertheless, in version 3 of CHART.JS was working because the types didn't expose the kurkle/color lib but the types were wrapped: https://cdn.jsdelivr.net/npm/chart.js@3.9.1/types/helpers/helpers.color.d.ts

Now, having the explicit import, I think (but not sure) you must add the dependency.

Reply:
what I cannot understand is why is working on a normal TS project:

https://codesandbox.io/s/typescript-forked-fb4e3s?file=/src/index.ts

Reply:
I guess the error is masked:
<img width="479" alt="image" src="https://user-images.githubusercontent.com/27971921/203632148-047c72b2-6142-4bbf-956f-cd89cff50c3e.png">

compared to this repo with @kurkle/color added as devDependency:
![image](https://user-images.githubusercontent.com/27971921/203632436-e1fb586f-d06e-44ce-870b-0e41a8b9a6b4.png)

Reply:
So, as a workaround, I think its ok to add `@kurkle/color` as devDependency here, until the issue is solved in chart.js.

Reply:
Or: https://github.com/chartjs/chartjs-plugin-zoom/pull/695/files/fcf885f159b2be3212b7a6cb03689820b51aea46..be9a095f957266f40a9495cfbae6aa840f02b443

Reply:
Believe me I was having a look to that commit in zoom. Maybe its better to try this parameter.
But just for my knowledge, do you think its something to fix in Chartjs?

Reply:
yes, I think so. It refers to a package for types and does not require that package to be installed. it either needs to bundle the types, not export them or require the dependency to be a nice citizen.

Reply:
Great!! Tomorrow I will apply the new parameter

Reply:
Done! Thank you @kurkle

Reply:
@stockiNail sure, go ahead and submit the issue (I was too lazy to do that)